src/Dropdown/Dropdown.js

```diff
@@ -395,23 +396,26 @@ class Dropdown extends Util.mixin(BindMixin) {
       );
     }
 
-    if (state.renderHidden) {
-      dropdownMenu = (
-        <div key="concealer" className="dropdown-menu-concealer">
-          {dropdownMenu}
-        </div>
-      );
-    }
+    // if (state.renderHidden) {
```

Comment:
@TattdCodeMonkey this is used to calc height of the list and then to decided whether it should open up or down. It's broken

Reply:
Figured this out. It is because the ref's were broken and not updated for React 16.

Reply:
oh well 😅